### ISSUE TITLE
About "logits = model.inference_refine(images, coarse, keep_conv, keep_hidden)"

### ISSUE THREAD

**Issue by hahapp** [MasazI/cnn_depth_tensorflow#12]
if REFINE_TRAIN:
            print("refine train.")
            coarse = model.inference(images, keep_conv, trainable=False)
            logits = model.inference_refine(images, coarse, keep_conv, keep_hidden)
        else:
            print("coarse train.")
            logits = model.inference(images, keep_conv, keep_hidden)

In these code, why use keep_conv and keep_hidden in inference_refine and inference?
Why reuse and trainable could be replaced by keep_conv and keep_hidden?
And, why keep_conv = 0.8 and keep_hidden = 0.5?

thanks,
